libs/muddle/include/muddle/punishment_broadcast_channel.hpp

```diff
@@ -115,6 +115,8 @@ class PunishmentBroadcastChannel : public service::Protocol, public BroadcastCha
   std::weak_ptr<core::Runnable> GetRunnable() override;
 
 private:
+  static constexpr char const *LOGGING_NAME = "PunishmentChannel";
```

Comment:
unrelated?

Reply:
Needed I removed the logging name from "Protocol" because this was being used as the default too often making the logging/debugging harder.

Reply:
Oh - that's an f up